Rakefile

```diff
@@ -168,11 +175,9 @@ namespace :acceptance do
   end
 
   desc 'Set up the test infrastructure'
-  task :setup do
+  task :setup, [:runner_platform] do
```

Comment:
I'd still like to be able to run the tests on a VM in case the machine I'm on is a PITA w/ containers. So, I think `acceptance:setup` should be something like `acceptance:setup[:vm|:container]`

Reply:
We can do that, but from a long term supportability perspective, all of these may have to go and the setup be refactored into the `spec_helper_acceptance_local` again. The issue is that if we use this kind of set up workflow, we have to set both the `Rakefile` and `.travis.yml` files to unmanaged in a `.sync.yml` file, which means that any changes that need to be made to these files to fix CI bugs, that would normally be easily implemented via a pdk sync would need to be manually evaluated and merged in. We don't feel this pain right now when we're writing a new module, but the modules team put a lot of effort into being able to standardize the module's ci files as much as possible because of the pain involved with maintaining them when they aren't standardized.

Reply:
I think "refactor setup to make long-term support easier" could be a separate issue and is out of scope for this PR. And when we do get to that stage, `:vm|:container` could be done some other way, e.g. as a `USE_CONTAINERIZED_PE_ENVIRONMENT` environment variable or something similar.

So, both-cases, I think this stuff should work with VMs and containers.

Reply:
Yup, out of scope for now is fine.